tasmota/tasmota.h

```diff
@@ -209,6 +209,12 @@ const uint32_t LOOP_SLEEP_DELAY = 50;       // Lowest number of milliseconds to
 #define KNX_MAX_device_param   31
 #define MAX_KNXTX_CMNDS        5
 
+// Switch Mode definietions
+#define SM_TIMER_MASK         0x3F
```

Comment:
Why are these constants declared in ```tasmota.h``` when only used by one source file?

Reply:
Fixed

Reply:
ty